installDep.sh

```diff
@@ -0,0 +1,7 @@
+cd node_modules/delphinus-curves
+npm install
```

Comment:
need change to npm run prepare, or it will duplicate install node_module locally.